rust/parquet/src/arrow/array_reader.rs

```diff
@@ -503,7 +492,14 @@ where
             data_buffer.into_iter().map(Some).collect()
         };
 
-        self.converter.convert(data)
+        // TODO: I did this quickly without thinking through it, there might be edge cases to consider
```

Comment:
This looks fine to me because this function originally returned this `Result`; I'm not sure what edge cases there might be...

Reply:
I wrote it in a hurry, so couldn't stop to think of there's anything I might miss. Happy that there isn't